examples/tutorials/Part 0 - Installation Guide.md

```diff
@@ -0,0 +1,228 @@
+
+# Part 0: Installing PySyft
+
+Before you start following the tutorials, you need to install PySyft on your computer.
+
+Here are the PySyft installation instructions for Linux, MacOs and Windows.
+
+Just scroll down to find your preferred OS:
+
+--------------
+
+Authors:
+
+- Alan Aboudib - Twitter: [@alan_aboudib](https://twitter.com/alan_aboudib)
+
+
+
+----------------------------
+## 0.1 Linux
+--------------------
+
+Installing PySyft on Linux is really straight forward. Here are the steps:
+
+#### 1. Make sure you have python >= 3.6
+
+You can check out the version number by running:
+
+<font color='red'>$</font>  `python --version`
+
+#### 2. Install pytorch 1.0:
```

Comment:
You dont need to install pytorch seperately. It is a dependency

Reply:
The thing is that the exact install command adapted to your  system might change as a function of what option you choose here? https://pytorch.org/get-started/locally/

I am not sure how `pip3 install -r requirements.txt` can handle this??

Reply:
Done!